plugins/com.google.cloud.tools.eclipse.sdk/src/com/google/cloud/tools/eclipse/sdk/CloudSdkManager.java

```diff
@@ -42,4 +48,41 @@ public static boolean isManagedSdkFeatureEnabled() {
     }
     return false;
   }
+
+  /**
+   * Installs a Cloud SDK, if the preferences are configured to auto-manage the SDK. Blocks callers
+   * 1) if the managed SDK is being installed concurrently by others; and 2) until the installation
+   * is complete.
+   *
+   * @param consoleStream stream to which the install output is written
+   */
+  public static void installManagedSdk(MessageConsoleStream consoleStream)
+      throws CoreException, InterruptedException {
+    if (isManagedSdkFeatureEnabled()) {
+      if (CloudSdkPreferences.isAutoManaging()) {
+        runInstallJob(consoleStream, new CloudSdkInstallJob(consoleStream));
```

Comment:
Does `runInstallJob()` actually buy us anything, or is it just for testing?

Reply:
Just for testing, unfortunately.